modules/plugin/geotiff/src/test/java/org/geotools/gce/geotiff/GeoTiffReaderTest.java

```diff
@@ -1310,6 +1313,32 @@ public void testRescaleWithNodataFloat() throws IOException {
         }
     }
 
+    @Test
+    public void testCOGRead() throws URISyntaxException, IOException {
+        String url =
```

Comment:
This is an online test, may not work all the time. Does image mosaic already have a "online" test category/profile?

Reply:
I have moved that test in a self contained XOnlineTest class so that it will be excluded when not using the online profile